dll/shellext/zipfldr/zipfldr.cpp

```diff
@@ -47,8 +48,15 @@ static void init_zlib()
     fill_win32_filefunc64W(&g_FFunc);
 }
 
+void Utf8ToWide(const CStringA& strUtf8, CStringW& strWide)
```

Comment:
MS's ATL has macros like `A2W` that we don't have:
https://learn.microsoft.com/en-us/previous-versions/visualstudio/visual-studio-2013/87zae4a3(v=vs.120)

Reply:
I will fix it later.